packages/fx-core/src/component/generator/constant.ts

```diff
@@ -10,7 +10,8 @@ export const defaultTimeoutInMs = 20000;
 export const sampleDefaultTimeoutInMs = 40000;
 export const templateFileExt = ".tpl";
 export const templateAlphaVersion = "templates-0.0.0-alpha";
-export const templateBetaVersion = "templates-0.0.0-beta";
+export const templateRcVersion = "templates@0.0.0-rc";
+export const templateV3Version = "^2.0.0";
```

Comment:
I think if we do not update template-config.json, how to download fallback zip files?